modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -161,17 +60,17 @@ export interface DOMUtilsSettings {
   keep_values: boolean;
   update_styles: boolean;
   root_element: HTMLElement | null;
-  collect: Function;
+  collect: boolean;
```

Comment:
This was apparently always a boolean, but in TinyMCE 5 it was changed from `collect: editor.inline` to `collect: () => inline` in `InitContentBody.ts`, see https://github.com/tinymce/tinymce/commit/da184ded52e9cb99226b151708bb67bbc0091fde. This meant it was always truthy when used below. As such, I put this back to the original behaviour but if this should always be truthy I'm happy to just add `collect: true` instead in `InitContentBody.ts`.